linux-armv7/Toolchain.cmake

```diff
@@ -11,9 +11,9 @@ set(CMAKE_Fortran_COMPILER $ENV{FC})
 set(CMAKE_CXX_FLAGS "-I ${cross_root}/include/")
 
 set(CMAKE_FIND_ROOT_PATH ${cross_root} ${cross_root}/${cross_triple})
-set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
-set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)
-set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
+set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM BOTH)
```

Comment:
@JonasVautherin I understand they others, and I see your comment in the issue, but it still seems that we should have `NEVER` here -- generally, `PROGRAM` is used to find a tool to run on the host system to help with the build. What use case do you have for setting `PROGRAM` to both? This will be problematic when a build finds a tool built for the target system, and it cannot be executed.

Reply:
If the `PROGRAM` is not in `PATH`, it is typically specified with `-D<helper-tool>_EXECUTABLE:FILEPATH=/a/host/filepath`.

Reply:
@JonasVautherin does this solve your use case?

Reply:
Yes, seems fine. Changing that. Thanks for the hint!